src/Trinity.Core/Network/Messaging/TrinityResponse.cs

```diff
@@ -35,7 +35,7 @@ static TrinityResponse()
 
         internal TrinityResponse(int size)
         {
-            Buffer = (byte*)CMemory.malloc((ulong)size);
+            Buffer = (byte*)CMemory.C_malloc((ulong)size);

```

Comment:
@leoxia for P/Invoke the C# signature should match the unmanaged, exported signature. But we cannot simply say DLL_EXPORT void* malloc(...) as this would clash with the libc implementation. The entry point can be specified in the DllImport attribute. But that creates inconsistency between the internal call entries, and the real entry names (say you want to find all occurrences of a signature, and inconsistency would probably cause it to fail)